### ISSUE TITLE
[BUG] Auctionator settings sometimes gets wiped to default

### ISSUE THREAD

**Issue by zeenk** [Auctionator/Auctionator#1163]
Please include as much of the following information as possible to help me fix the bug:

## Brief Description of Bug
Logged in, got lua error, opened AH and my settings for Auctionator were reset to default, had to use an old backup to get them back. (Using Version: 9.2.17)
![2022-05-29_21-48-14](https://user-images.githubusercontent.com/68170439/170888867-c6ecdc99-ca03-4aa1-a46f-a7b6ff33df31.png)

## Steps to Reproduce
Have not found a real way to reproduce the bug, just randomly happens sometimes when I logon my 20 different realms to repost auctions and checking for sniper deals, started happening last week. (Does not happen super often)

Maybe has something to do with Point Blank Sniper?

## Loaded Addons
Advanced Interface Options
AllTheThings
Angry Keystones
Ansi Combat Text
Auctionator
BagBrother
Bagnon
BetterWardrobe
BigDebuffs
Can I mog it?
Community to raid
Details!
Handynotes
Hidingbar
In Progress Missions
Kui Nameplates
Leatrix Plus
Leatrix Maps
Maze Helper
MogPartialSets
Mount Journal Enhanced
MoveAnything
Myslot
OmniCC
Opie
Oribos Exchange
Paste
Pawn
Point Blank Sniper
Postal
Prat 3.0
Pretty Reps
Quartz
RaiderIO
Rarity
Rrematch
SavedInstances
Silverdragon
Simulationcraft
TL;DR Missions
TomCat's Tours
TomTom
Venture Plan
WeakAuras
World Quest List
Z-Perl
Zereth Mortis Puzzle Helper
TradeSkillMaster
TradeSkillMaster_AppHelper
Premade Sort
DeadlyBossMods
DejaCharacterStats
Twitch Emotes

## Back Trace (LUA Error Output)
I was getting these lua errors but it doesn't say it's for Auctionator specifically but it's the only addon that had settings wiped.
![2022-05-29_21-38-31](https://user-images.githubusercontent.com/68170439/170888662-a8efbffd-a591-4d8c-b98f-84ee9b11bf5b.png)
![2022-05-29_21-38-47](https://user-images.githubusercontent.com/68170439/170888667-d770cea9-ddce-4287-b02e-7f935c152327.png)

**Comment by zeenk** [comment]
Just happened again and noticed in the saved variables that the Auctionator.lua was pretty large compared to the other ones 🤔 

![2022-05-30_00-17-47](https://user-images.githubusercontent.com/68170439/170893500-193ef12a-e230-4a19-b186-7b85743efff9.jpg)

**Comment by zeenk** [comment]
I deleted the old auctionator saved variable lua file and now it's looking a lot better, I'll reopen this issue if I run into this issue again.

![2022-05-30_01-28-13](https://user-images.githubusercontent.com/68170439/170895566-34b2ffe4-b985-41c7-8fc4-16f17a04d21b.png)

**Comment by plusmouse** [comment]
This usually happens when scanning on a lot of realms. You can avoid this issue by running `/atr nopricedb`, which will disable the tooltip price and price histories (which is what causes the glitch).

**Comment by zeenk** [comment]
I never use the AH scan feature for Auctionator I'm just using the Point Blank Sniper scanner, but it's probably the same thing I guess? except it only scans for the items in the Starter shopping list, and I kinda want to keep the price history feature because it's pretty handy for me since I post on a few low pop realms where I have no competition so when the auctions expire I can just click on the last history price for the item instead of manually have to type the numbers in. :P
